modules/nw-throughput-troubleshoot.adoc

```diff
@@ -0,0 +1,53 @@
+// Module filename: nw-throughput-troubleshoot.adoc
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id='nw-throughput-troubleshoot-{context}']
+= Troubleshooting Throughput Issues
+Sometimes applications deployed through {product-title} can cause
+network throughput issues such as unusually high latency between
+specific services.
+
+Use the following methods to analyze performance issues if pod logs do not
```

Comment:
All instances of "pod" should be "Pod" (when used in a sentence, not a command). Can you do a sweep through all the files in this PR and update?